habitat-baselines/habitat_baselines/il/data/nav_data.py

```diff
@@ -50,7 +61,7 @@ def __init__(
         """
         self.config = config.habitat
         self.env = env
-        self.episodes = self.env._dataset.episodes  # type:ignore
+        self.episodes: List[NavigationEpisode] = self.env._dataset.episodes
```

Comment:
@vincentpierre Was this part of the code checked? I am not sure it's covered by tests and I worry it could error on non-typechecking runtime?

Reply:
Sorry I don't understand. I did not add any unit tests for il in this PR. Why do you think it might break ?

Reply:
Because NavigationEpisode is not imported if not TYPE_CHECKING so the interpreter will break when it gets to that line, no?

Reply:
Even if it is in quotes?

Reply:
```
python -c "from habitat_baselines.il.data.nav_data import NavDataset
class A:
   habitat = 1
class B:
   episodes = 2
class C: 
   _dataset = B()
n = NavDataset(A(),C(),3)
"
```
Breaks line 67 instead of 53, so I guess there is no issue.

Reply:
> Even if it is in quotes?

Missed that it was in quotes. Well if it didn't break anything obvious then it should be fine.